hoodie-common/src/main/avro/HoodieCommitMetadata.avsc

```diff
@@ -92,6 +92,11 @@
                         "name":"totalRollbackBlocks",
                         "type":["null","long"],
                         "default" : null
+                     },
+                     {
+                        "name":"fileSizeInBytes",
```

Comment:
why is this needed for this change? to provide file length as a part of FileStatus api?